tracing-opentelemetry/src/layer.rs

```diff
@@ -364,11 +365,21 @@ where
     {
         OpenTelemetryLayer {
             tracer,
+            span_timings_enabled: self.span_timings_enabled,
             get_context: WithContext(OpenTelemetryLayer::<S, Tracer>::get_context),
             _registry: self._registry,
         }
     }
 
+    /// Sets whether or not spans metadata should include the _busy time_ (total time for which it was entered),
```

Comment:
nit: can we wrap these comments as well?

Reply:
Apologies, but what do you mean by wrapping theses comments?

Reply:
Oh my bad, you meant wrapping it to col:80. Updating.

Reply:
ah, have them wrap to the next line to match the other comments (after 80 characters)